declarations/WebpackOptions.d.ts

```diff
@@ -1205,6 +1206,10 @@ export interface OutputOptions {
 	 * If `output.libraryTarget` is set to umd and `output.library` is set, setting this to true will name the AMD module.
 	 */
 	umdNamedDefine?: boolean;
+	/**
+	 * The filename of URL modules as relative path inside the `output.path` directory.
+	 */
+	urlModuleFilename?: string;
```

Comment:
/cc @hiroppy Based on `file-loader` history it should accept `Function` also. We allow people use `Function` in `output.filename` (https://webpack.js.org/configuration/output/#outputfilename).  

Maybe we need investigate all tests in file-loader and move their here?

What do you think?

Reply:
Yes, We need to implement features that can be used with the file/url-loader.
So, I want to implement it in the next step. Also, I want to move all test cases from these loaders.

Reply:
:+1: Great